app/controllers/experiments/list.js

```diff
@@ -107,9 +106,19 @@ export default Ember.Controller.extend({
                     sequence: []
                 }
             });
-            this.send('toggleModal');
-            newExperiment.save().then(function() {
-                self.transitionToRoute('experiments.info', newExperiment.id);
+            newExperiment.save();
```

Comment:
But the event listener should be setup before triggering the event, right? Or am I missing something here.